### PR TITLE
Replace gomigrate by tern in tools

### PR DESCRIPTION
**Pull request by ezr-ondrej**
@lzap this is not needed now, right?

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -47,7 +47,6 @@ install-tools:
 	go install golang.org/x/tools/cmd/goimports@latest
 	# pin for a bug: https://github.com/golangci/golangci-lint/issues/2851
 	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.45.2
-	go install -tags 'postgres' github.com/golang-migrate/migrate/v4/cmd/migrate@latest
```
Comment (lzap) [inline Makefile:50]: Can you replace this with

    go install github.com/jackc/tern@latest

instead? Actually the tern CLI is quite useful and we can use it, just add `tern.conf` into .gitignore and people can configure their database and then issue the tern command to operate on it (migrate, get status, create new migration file).

File: Makefile [hunk 2]
```diff
@@ -32,7 +32,7 @@ run:
 .PHONY: update-clients
 update-clients:
 	wget -O ./configs/ib_api.yaml -qN https://raw.githubusercontent.com/osbuild/image-builder/main/internal/v1/api.yaml
-	wget -O ./configs/sources_api.json -qN https://raw.githubusercontent.com/RedHatInsights/sources-api-go/main/public/openapi-3-v3.1.json 
+	wget -O ./configs/sources_api.json -qN https://raw.githubusercontent.com/RedHatInsights/sources-api-go/main/public/openapi-3-v3.1.json
```
Comment (lzap) [inline Makefile:35]: I have that in my other PR just drop it if you can.

File: Makefile (hunk 1 above)
Comment (ezr-ondrej) [reply Makefile:50]: sure :)

**Review by lzap — APPROVED** [review]
Yeah, that's useful. Thanks.

**Pull request merged**